apps/api/src/configs/app.config.ts

```diff
@@ -22,11 +22,16 @@ export const appConfigSchema = Joi.object({
   APP_PORT: Joi.number().default(4000),
   APP_ADDRESS: Joi.string().default('0.0.0.0'),
   BASE_URL: Joi.string().required(),
+  ENABLE_AUTO_FEEDBACK_DELETION: Joi.boolean().default(false),
+  AUTO_FEEDBACK_DELETION_PERIOD_DAYS: Joi.number().default(365 * 5),
```

Comment:
How about using when() to make it optional required?

```ts
AUTO_FEEDBACK_DELETION_PERIOD_DAYS: Joi.number().when('ENABLE_AUTO_FEEDBACK_DELETION', {
  is: true,
  then: Joi.required(),
  otherwise: optional(),
})
```